src/basic_recipes/pie.jl

```diff
@@ -17,14 +19,27 @@ Creates a pie chart from the given `values`.
     inner_radius = 0
     "The angular offset of the first pie segment from the (1, 0) vector in radians."
     offset = 0
+    "The offset of each pie segment from the center along the radius"
+    offset_radius = 0
     MakieCore.mixin_generic_plot_attributes()...
 end
 
-function plot!(plot::Pie)
+convert_arguments(PT::Type{<:Pie}, values::RealVector) = convert_arguments(PT, 0.0, 0.0, values)
+convert_arguments(PT::Type{<:Pie}, point::VecTypes{2}, values::RealVector) = convert_arguments(PT, point[1], point[2], values)
+convert_arguments(PT::Type{<:Pie}, ps::AbstractVector{<:VecTypes{2}}, values::RealVector) = convert_arguments(PT, getindex.(ps, 1), getindex.(ps, 2), values)
+convert_arguments(::Type{<:Pie}, xs::Union{Real,RealVector}, ys::Union{Real,RealVector}, values::RealVector) = begin
+    xs = length(xs) == 1 ? fill(only(xs), length(values)) : xs
+    ys = length(ys) == 1 ? fill(only(ys), length(values)) : ys
+    return (float_convert(xs), float_convert(ys), float_convert(values))
+end
```

Comment:
I would prefer this to be a `function` rather than a shorthand with `begin ... end`. Otherwise I'm happy with this

Reply:
updated. thanks